R/pkg/R/generics.R

```diff
@@ -651,23 +651,25 @@ setGeneric("write.jdbc", function(x, url, tableName, mode = "error", ...) {
 
 #' @rdname write.json
 #' @export
-setGeneric("write.json", function(x, path) { standardGeneric("write.json") })
+setGeneric("write.json", function(x, path, mode = NULL, ...) { standardGeneric("write.json") })
```

Comment:
would it be better as more generic to have these generics as `setGeneric("write.json", function(x, path, ...)` instead?